client/blocks/product-purchase-features-list/google-vouchers.jsx

```diff
@@ -5,18 +5,30 @@
  */
 
 import React from 'react';
+import { useTranslate } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import GoogleVoucherDetails from 'my-sites/checkout/checkout-thank-you/google-voucher';
+import PurchaseDetail from 'components/purchase-detail';
 import QuerySiteVouchers from 'components/data/query-site-vouchers';
 
 export default ( { selectedSite } ) => {
+	const translate = useTranslate();
 	return (
 		<div className="product-purchase-features-list__item">
 			<QuerySiteVouchers siteId={ selectedSite.ID } />
-			<GoogleVoucherDetails selectedSite={ selectedSite } />
+			<PurchaseDetail
+				alt=""
+				id="google-credits"
+				icon={ <img alt="" src="/calypso/images/illustrations/google-adwords.svg" /> }
+				title={ translate( 'Google Ads credit' ) }
+				description={ translate(
+					'Use a $100 credit with Google to bring traffic to your most important Posts and Pages.'
```

Comment:
What do you think of extracting the amount from this string? That way there won't be any additional work for translators the day we decide to change it.

Reply:
Done